google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/instance/AppEngineServerModel.java

```diff
@@ -162,20 +165,28 @@ public void checkConfiguration() throws RuntimeConfigurationException {
           "App Engine facet not found in '" + artifact.getName() + "' artifact");
     }
 
-    if (CloudSdkService.getInstance().getSdkHomePath() == null) {
+    CloudSdkService cloudSdkService = CloudSdkService.getInstance();
+    if (cloudSdkService.getSdkHomePath() == null
+          || StringUtils.isEmpty(cloudSdkService.getSdkHomePath().toString())) {
```

Comment:
They should be different. If the SDK location is empty, is because the user set it that way. In that case, we should tell them No SDK was found in that directory, rather than an SDK directory wasn't specified.